app/src/main/res/layout/prompt_color.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/promptColorLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@drawable/prompt_background">
+
+        <TextView
+            android:id="@+id/title"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_horizontal"
+            android:textSize="24sp"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toTopOf="@id/colorPickerView"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:text="ColorPicker Dialog" />
+
+        <com.skydoves.colorpickerview.ColorPickerView
+            android:id="@+id/colorPickerView"
+            android:layout_width="325dp"
+            android:layout_height="325dp"
+            android:layout_margin="10dp"
+            app:layout_constraintBottom_toTopOf="@id/buttonsLayout"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/title"
+            app:palette="@drawable/palette"
+            app:preferenceName="MyColorPicker" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/buttonsLayout"
+            android:layout_width="wrap_content"
```

Comment:
One last thing: this line needs to be replaced with:

```
android:layout_width="match_parent"
```

This change aligns the buttons to the right of the dialog window, like in other similar prompts.

For example, this is how the confirmation prompt looks like now:

![com igalia wolvic-20230413-154306](https://user-images.githubusercontent.com/1412060/231676613-03574ea0-ccac-49ba-b35c-acd6f2c0f578.jpg)

And this is how the color picker prompt will look with the suggested change:

![com igalia wolvic-20230413-154236](https://user-images.githubusercontent.com/1412060/231676679-d19d40e6-c0a0-4c20-a94c-56c2721ebcfa.jpg)

Reply:
Sure will do 👍
Thanks !😊

Reply:
Thank you!